tests/testthat/test-expect-known.R

```diff
@@ -12,12 +12,9 @@ test_that("uses specified width", {
 
 test_that("creates file on first run", {
   file <- withr::local_tempfile()
-  expect_success(
-    expect_warning(
-      expect_known_output(cat("ok!\n"), file),
-      "Creating reference"
-    )
-  )
+  expect_known_output(cat("ok!\n"), file) |>
```

Comment:
the 4.1.0 requirement is so satisfying 🧘‍♂️

Reply:
The simple joys of being about to use R features that were released 5 years ago 😆